### ISSUE TITLE
Individual column filtering

### ISSUE THREAD

**Issue by mke21** [shymonk/django-datatable#26]
Is it possible to implement filtering/searching on individual columns?
Like:
https://datatables.net/examples/api/multi_filter.html

**Comment by ikenfin** [comment]
Hi! I added basic support for this feature in my fork, please check this https://github.com/ikenfin/django-datatable.

**Comment by shymonk** [comment]
Is it possible to make a pull request?
New feature is not only for me, but also for other contributors.

**Comment by ikenfin** [comment]
@shymonk Yes it's possible, but before adding this feature, source code was converted by `2to3`, so there is some extra code in my fork. If it's okay, then i will PR, or i can create new fork and add only column filtering.

**Comment by shymonk** [comment]
Create a new fork and add only column filtering is more simple and clear. For `2to3` support, please make another PR.

**Comment by mke21** [comment]
@shymonk, will this feature be implemented in the main branch?

**Comment by ikenfin** [comment]
@mke21 don't think so - github says that there some conflicts between branches, and i, unfortunately don't have time to fix the code at now.
